packages/core/src/data-access/documentVersions/getDocumentsAtCommit.ts

```diff
@@ -91,34 +91,3 @@ export async function getDocumentsAtCommit(
 
   return Result.ok(totalDocuments)
 }
-
-export async function getDocument({
-  projectId,
-  commitUuid,
-  documentId,
-}: {
-  projectId: number
-  commitUuid: string
-  documentId: number
-}): Promise<TypedResult<{ content: string }, LatitudeError>> {
-  const commitResult = await findCommit({ uuid: commitUuid, projectId })
-  if (commitResult.error) return commitResult
-  const commit = commitResult.unwrap()
-
-  const result = await database
-    .select({ content: documentVersions.content })
-    .from(documentVersions)
-    .where(
-      and(
-        eq(documentVersions.id, documentId),
-        eq(documentVersions.commitId, commit.id),
-      ),
-    )
-
-  if (result.length === 0) {
-    return Result.error(new NotFoundError('Document not found'))
-  }
-
-  const documentVersion = result[0]!
-  return Result.ok({ content: documentVersion.content ?? '' })
-}
```

Comment:
this method was unnecessary complex? document id is already a unique identifier we don't need to fetch by projectid nor commit id.

Reply:
How you ensure tenancy without checking `project` or `commit`?

Reply:
uff.. are we establishing tenancy at data accessors? in any case tenancy is at workspace level, not project or commit level, so it would mean all accessors need to receive workspace

Reply:
> are we establishing tenancy at data accessors?

We have the repositories in the old code. Was nice but I feel it might be harder with Drizzle

Reply:
hmm we will require a bigger refactor to solve tenancy across all data accessors. I'll give it a try in another PR.

Reply:
Nice!